edenia/antelope-tools
# Feat/evm dashboard frontend 1244

### Telos EVM Dashboard

### What does this PR do?

- Resolve #1244 
- Add EVM Dashboard and EVM RPC endpoints pages
- Refactor cards and health status components
- Fix tps and tpb in mainnet networks

### Steps to test

1. Run the project locally with EVM configuration
2. Go to EVM Dashboard page
3. Check the transactions per block chart and the stats data
4. Go to EVM RPC Endpoints page 
5. Run the test and check the results

### Screenshots

# EVM Dashboard

![imagen](https://github.com/edenia/antelope-tools/assets/66583677/e7c2db86-e97b-4cfc-8632-a677f00d9562)
![imagen](https://github.com/edenia/antelope-tools/assets/66583677/f84a6db2-4fe0-48a9-be16-481a72de7931)

# EVM RPC Endpoints

![imagen](https://github.com/edenia/antelope-tools/assets/66583677/5c294185-bafc-4454-9e13-a67f3525a6f6)
![imagen](https://github.com/edenia/antelope-tools/assets/66583677/1276645a-b2ea-4495-9b1f-99226220ccca)
![imagen](https://github.com/edenia/antelope-tools/assets/66583677/27f6ea8d-3ca9-4f5d-b9e2-cee6eb9f935c)
![imagen](https://github.com/edenia/antelope-tools/assets/66583677/7ec1281e-b976-477a-b434-2d7242b6c7bc)
